### PR TITLE
fix token accounts liquidity pool

### PR DESCRIPTION
**Pull request by BouweCeunen**

### CODE REVIEW

File: src/credix.json [hunk 1]
```diff
@@ -525,8 +570,5 @@
       "name": "InterestRepaid",
       "msg": "Interest is already repaid."
     }
-  ],
-  "metadata": {
-    "address": "B7PiFKNiBvQPMVtsJt8bM86U69a1ivev4VvnkLViMiUZ"
-  }
```
Comment (BouweCeunen) [inline src/credix.json:531]: my anchor build does not include this anymore it seems

File: src/store/api.ts [hunk 2]
```diff
@@ -190,6 +185,32 @@ const getAssociatedUSDCTokenAddressPK = multiAsync(
...
+	async (connection: Connection, wallet: Wallet) => {
+		const lpTokenMintPK = await getLPTokenMintPK(connection, wallet);
+		return Token.getAssociatedTokenAddress(
+			ASSOCIATED_TOKEN_PROGRAM_ID,
+			TOKEN_PROGRAM_ID,
+			lpTokenMintPK,
+			wallet.publicKey
+		);
+	}
+);
```
Comment (BouweCeunen) [inline src/store/api.ts]: relocated this to keep all associated token accounts things together

File: src/store/api.ts [hunk 3]
```diff
@@ -190,6 +185,32 @@ const getAssociatedUSDCTokenAddressPK = multiAsync(
...
+
+const getAssociatedLiquidityPoolUSDCTokenAddressPK = multiAsync(
```
Comment (RuvendeGroote) [inline src/store/api.ts]: is this the same account as stored in the global market state?

Comment (RuvendeGroote) [reply src/store/api.ts]: we already have a function to get this account so this one is not needed.
https://github.com/credix-finance/credix-app/pull/40/files#diff-3c19dac050eb3dce614baee16edf7f6b60a73b83307660b84f7650e1347adacdR166

**Review by RuvendeGroote — APPROVED** [review]

**Pull request merged**